dev/build-arrow-ballista.sh

```diff
@@ -15,8 +17,18 @@
 # specific language governing permissions and limitations
 # under the License.
 
-# the default configuration location is "/etc/ballista/scheduler.toml"
-# if you include a specifc conf file using "--config-file = my_config_file.toml"
-# then that file will override environment variables, but not command line arguments
-namespace = "my_name_space"
-bind_host = "1.2.3.4"
\ No newline at end of file
+set -e
+
+# delete any previously cloned arrow-ballista repo
+rm -rf arrow-ballista 2>/dev/null
+
+# clone the repo
+# TODO make repo/branch configurable
+git clone https://github.com/apache/arrow-ballista
```

Comment:
Do we want to always pin this so that Ballista changes can't side-effect on unrelated DataFusion PRs?

Reply:
That's a good question and I'm really not sure what the best approach is. Perhaps we wait until we hit this situation and then we can pin to the commit prior to the breaking change.

DataFusion no longer depends on Ballista so I am not sure what kind of change would cause us to deal with this situation but I am sure it is possible somehow.